src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--29">
+      <div class="calendar__day calendar__day--1 calendar__day-start--sun"></div>
+      <div class="calendar__day calendar__day--2"></div>
```

Comment:
What if we remove classes from each div? Can we do it without them?

Reply:
I think it's impossible, because this classes need for @for and @each loop